extensions/ql-vscode/src/vscode-tests/cli-integration/data/simple-query.ql

```diff
@@ -1,3 +1,5 @@
+import javascript
```

Comment:
Does this break any existing CLI tests? If not, then this is fine, but I would think that the other tests that try to use this file might have a problem resolving the javascript import.

Reply:
> Does this break any existing CLI tests?

It does, sadly 🙃 I'll try using a separate test query...